src/vl-map.js

```diff
@@ -57,16 +57,25 @@ export class VlMap extends VlElement(HTMLElement) {
         return this._shadow.querySelector('#map');
     }
 
-    get _projection() {
-        return new ol.proj.Projection({
-            code: 'EPSG:31370',
-            extent: this._extent
-        });
-    }
-
-    get _extent() {
-        return [9928, 66928, 272072, 329072];
-    }
+  get _porjectionCode() {
+    return 'EPSG:31370';
+  }
+
+  get _projection() {
+    return new ol.proj.Projection({
+      code: this._porjectionCode,
+      extent: this._extent
+    });
+  }
+
+  convertWGS84(longitude, latitude) {
```

Comment:
@Pieter-Beckers is het niet beter om de projectie configureerbaar te maken en EPSG:4326 te definiëren? Nu gaan jullie vermoedelijk alles omzetten naar EPSG:31370 m.b.v. de convertWGS84 functie, maar dat is overbodig als de projectie goed staat?

Reply:
Ik denk dat dit niet de beste plek is om zo'n util functie te zetten. Wij hebben van de map eigenlijk enkel de projectionCode nodig.

Reply:
Inderdaad. Ik zou het zelfs zo doen dat de projection code meegegeven kan worden als attribuut. Ik kan niet meteen een use case bedenken dat nadien de projection code gewijzigd zou moeten worden en hiervoor is ook een code uitbreiding nodig.

Voorstel: data-vl-projection-code attribuut introduceren met default EPSG:31370 als waarde indien dit attribuut niet gedefinieerd is.

Reply:
Dit heeft niet veel waarde... De projectioncode zelf is maar een symbol. Wij moet nog altijd de coordinate system definiëren en de extent koppelen. 

Ik stel nu gewoon de bestaand get _projection() methode public maken.

Reply:
Op zich zouden we er zelfs vanuit kunnen gaan dat het default altijd EPSG:31370 en dit in ons project ook aannemen. Maar hier in dit issue het projectiesysteem dynamisch maken zou ik ook niet direct doen (hier is niet direct vraag naar).

Reply:
@Pieter-Beckers ticket en PR dus afsluiten zonder gevolg?

Reply:
@Pieter-Beckers ticket en PR dus afsluiten zonder gevolg?

Reply:
@Pieter-Beckers ticket en PR dus afsluiten zonder gevolg?

Reply:
Ja, mag weg. Code is verplaatst naar het project zelf.